src/expr/proof_checker.cpp

```diff
@@ -107,6 +107,11 @@ Node ProofChecker::check(
     Assert(expected.isNull() || expected == args[0]);
     return expected;
   }
+  if (id == PfRule::VERIT_RULE)
+  {
+    Assert(args.size() > 1);
```

Comment:
```suggestion
    Assert(args.size() > 1 && args[0].getType().isReal());
```
Actually it'd be better to check "isInteger" but I'm not sure that by creating a rational constant this will work. You could try both?

Reply:
isInteger works.